### ISSUE TITLE
[Prefs] the prefs shim is not working with local storage

### ISSUE THREAD

**Issue by jasonLaster** [devtools-html/debugger.html#1364]
I'm seeing some issues with the prefs shim locally. I'll take a look in a bit.

STR:
1. create a new pref `foo` with the prefs helper (see below)
2. in the console do `prefs.foo = false`
3. refresh
4. in the console `prefs.foo`

you expect to see false, but get true and if you screenshot localstorage you get

![screen shot 2016-11-30 at 3 11 19 pm](https://cloud.githubusercontent.com/assets/254562/20770401/9a86d18c-b713-11e6-881b-7becdfa59e3c.png)

```diff
diff --git a/src/utils/prefs.js b/src/utils/prefs.js
index 93e385b..9d7ed96 100644
--- a/src/utils/prefs.js
+++ b/src/utils/prefs.js
@@ -4,10 +4,14 @@ const { isDevelopment } = require("devtools-config");

 if (isDevelopment()) {
   pref("devtools.debugger.client-source-maps-enabled", true);
+  pref("devtools.debugger.foo", true);
 }

 const prefs = new PrefsHelper("devtools", {
   clientSourceMapsEnabled: ["Bool", "debugger.client-source-maps-enabled"],
+  foo: ["Bool", "debugger.foo"],
 });

+window.prefs = prefs;
+
 module.exports = { prefs };
```

CC @bomsy

**Comment by juliandescottes** [comment]
@jasonLaster can this be closed now that #1428 was merged?

**Comment by jasonLaster** [comment]
Yep.
